app/src/main/java/com/example/android/lifecycles/step5_solution/Fragment_step5.java

```diff
@@ -74,13 +79,14 @@ public void onStopTrackingTouch(SeekBar seekBar) { }
         });
 
         // Update the SeekBar when the ViewModel is changed.
-        mSeekBarViewModel.seekbarValue.observe(getActivity(), new Observer<Integer>() {
-            @Override
-            public void onChanged(@Nullable Integer value) {
-                if (value != null) {
-                    mSeekBar.setProgress(value);
-                }
-            }
-        });
+        mSeekBarViewModel.seekbarValue.observe(
+                requireNonNull(getActivity()), new Observer<Integer>() {
```

Comment:
Can use `requireActivity()` instead of `requireNonNull(...)`

Reply:
done